### ISSUE TITLE
groups as objects

### ISSUE THREAD

**Issue by kesla** [marcusberner/sandal#5]
Yo!

It would be pretty convenient I think if you got groups as objects, so that the following would be possible:

``` js
sandal.object('beep', {foo: 'bar'}, ['group']);
sandal.object('boop', {hello: 'world'}, ['group']);

sandal.resolve(function (err, group) {
  console.log('beep', group.beep);
  console.log('boop', group.boop);
});
```

what do you think?

**Comment by marcusberner** [comment]
It has now been implemented in v2.0.0. Thanks for great input!

**Comment by kesla** [comment]
@marcusberner Fantastic. Thanks!
